tutorials/plot_sensors_time_frequency.py

```diff
@@ -87,7 +87,8 @@
 # but you can also use :func:`mne.time_frequency.tfr_multitaper`
 # or :func:`mne.time_frequency.tfr_stockwell`.
 
-freqs = np.arange(6, 30, 3)  # define frequencies of interest
+# define frequencies of interest (log-spaced)
+freqs = np.logspace(*np.log10([6, 35]), num=8)
```

Comment:
Right, I didn't notice that. I thought pcolormesh will handle it right, but I'll set the ylim to the correct values.
Regarding `yticks` - what would you suggest? Checking number/location of ticks and relocating them to closest frequency bins? I'm afraid not everything will work smoothly with many non-uniform freqs (the tutorial has only 8 freq bins).